site/components/footer/index.js

```diff
@@ -198,12 +193,14 @@ const Footer = () => (
       <div className={styles.footerEndContainer}>
         <div className={cx('section', 'disclaimer', 'noBorder')}>
           <p className={cx('afterDivider', 'disclaimerParagraph')}>
-            &copy; Red Badger Consulting Limited 2016
+            © Red Badger Consulting Limited 2016
           </p>
           <p className={cx('afterDivider', 'disclaimerParagraph')}>
             Registered in England No. 7242017
           </p>
-          <p className={styles.disclaimerParagraph}>VAT Registration No. 990 8085 82</p>
+          <p className={styles.disclaimerParagraph}>
+            VAT Registration No. 990 8085 82
+          </p>
           <p className={styles.cookieWarning}>
             We use cookies on our website. For more information, view our privacy policy.
```

Comment:
It would be best to run prettier in bulk so the diffs don't show up in PRs. Can we do this after merge?